fastlane/Fastfile

```diff
@@ -38,80 +67,149 @@ platform :ios do
     # For example, if it is 15, you should set in xcode build number 15 (because increment_build_number increments the build).
     increment_build_number
 
+    # Bundle identifier from xCode project
+    bundle_identifier = read_bundle_identifier(
+      build_configuration: build_configurations[:testflight]
+    )
+
     # Build
     build_app(
-      # Set the app id
-      app_identifier: ENV["APP_IDENTIFIER"],
-      # What configuration to use, usefull for keeping different API keys etc between environments
-      configuration:"Release",
-      # Use this codesigning identity (this is the name of the certificate in your keychain)
-      codesigning_identity: ENV["CODE_SIGNING_ENTITY"],
-      # the projectname, this is the name of the .xcodeproj file and the folder containing your code in the project
-      project_name: ENV["PROJECT_NAME"],
-      # the scheme to build
-      scheme: "",
-      # Your team ID
-      team_id: ENV["TEAM_ID"]
+      app_identifier: bundle_identifier,
+      configuration: build_configurations[:testflight],
+      match_type: match_types[:testflight],
+      project_name: ENV["PROJECT_NAME"]
     )
 
-    upload_dsym_to_rollbar(
-      rollbar_url: 'https://api.rollbar.com/api/1/dsym',
-      access_token: ENV["ROLLBAR_POST_SERVER_ITEM_ACCESS_TOKEN"],
-      version: get_info_plist_value(path: '', key: 'CFBundleVersion'), # complete with info.plist path. ex: 'Project/Info.plist'
-      bundle_identifier: ENV["APP_IDENTIFIER"],
-      dsym: '' # DSYM location with @. Example: '@../wolox.app.dSYM.zip'. @ means that it takes the content of the file.
+    # Get rollbar server access token from configuration file.
+    rollbar_server_access_token = read_xcconfig_property(
+      xcconfig_path: project_xcconfig_path % [ENV['PROJECT_NAME'], build_configurations[:testflight]],
+      xcconfig_key: 'ROLLBAR_SERVER_ACCESS_TOKEN'
+    )
+
+    # Upload dsym to rollbar.
+    upload_dsym(
+      access_token: rollbar_server_access_token,
+      version: get_build_number,
+      bundle_identifier: bundle_identifier,
+      dsym_zip_path: dsym_zip_path % ENV['PROJECT_NAME']
     )
 
     publish_testflight
+  end
+
+  lane :test do
+    # Run scan with default project and scheme
+    scan(
+      configuration: build_configurations[:test],
+      clean: false
+    )
 
   end
 
-  private_lane :build_app do |options|
+  lane :create_development_app do
+    # Remember after this point to choose this profile in xCode Signing (Development)
+    create_app(
+      app_name: app_names[:development] % ENV['PROJECT_NAME'],
+      build_configuration: build_configurations[:development],
+      team_name: ENV['TEAM_NAME'],
+      skip_itc: true,
+      match_type: match_types[:development],
+    )
+
+    # Remember after this point to choose this profile in xCode Signing (Alpha)
+    create_app(
```

Comment:
This lane `create_app` doesn't build the app, it creates the app id and provisioning profile in member center. This is supposed to be run only once during the kickoff and create both application we'll use for development.

I know we usually don't create an app for development, but this is not the best, we don't do it just because it's not automatic. We should always create an app for development, as required from iOS 10. Also, there are services like push notifications that require it too.